list_comprehension_exercises.py

```diff
@@ -0,0 +1,19 @@
+from typing import List
+
+
+def all_numbers_form_one_to_thousand_divisible_by_seven():
+    original_list = list(range(1, 1001))
+    return [number for number in original_list if number % 7 == 0]
+
+
+def all_numbers_form_one_to_thousand_with_three_in_them():
+    original_list = list(range(1, 1001))
+    return [number for number in original_list if "3" in str(number)]
+
+
+def count_the_number_of_spaces_in_string(sentence: str) -> int:
+    return len([space for space in sentence if space == " "])
+
+
+def list_all_consonants_in_sentence(sentence: str) -> List:
+    return list([letter for letter in sentence if letter not in "aeiouAEIOU "])
```

Comment:
Should there be space in the end?

Reply:
Yes, becouse task was to count only consonants in given sentence. So function was counting consonants and free spaces as different chars. Space at the end, is then like vowel

Reply:
Kk, makes sense

Reply:
What if there are numbers of other characters then? :*

Reply:
I think error. But in this level of exercise it wasn't case. I have chosen this solution becouse it was faster than writing all consonants small and capital. This would be my solution if exercise would be more complicated.